src/components/atoms/button/button.scss

```
@@ -1,28 +1,30 @@
+@use "../../variables" as *;
 .text-padding {
-  background-color: black;
+  background-color: $black;
   position: relative;
   transition: 0.2s;
+  width: 200px;
+  height: 55px;
 
-  @media screen and (min-width: 576px) {
+  @media screen and (min-width: $breakpoint-sm) {
     width: 180px;
     height: 45px;
   }
-  @media screen and (min-width: 768px) {
+  @media screen and (min-width: $breakpoint-md) {
     width: 150px;
     height: 55px;
   }
-  @media screen and (min-width: 992px) {
+  @media screen and (min-width: $breakpoint-lg) {
     width: 200px;
     height: 55px;
   }
 }
 
 a {
   font-family: "Noto Sans JP";
```

https://github.com/shinonome-inc/-fukumitsu-Fukumitsu-Yuki--frontend-basic-last-assignment/blob/e20d1e3cc6efad1e4b507df109f034ca18d1033b/src/components/utilities.scss#L3-L5

`utilities.scss` で fontfamily を `body` 要素に対して設定しています。
CSSの[継承](https://developer.mozilla.org/ja/docs/Web/CSS/inheritance)という仕組みにより、`body` の中に入っている要素にはすべて `Noto Sans JP` が適用されています。
そのため、基本的にそれぞれの要素に対してゴシック体の指定は不要です。

また、設定する場合は以下のように変数を使いましょう！
```suggestion
  font-family: $fontfamily-gothic;
```